src/ontology/slurp/ordo.tsv

```diff
@@ -0,0 +1,329 @@
+mondo_id	xref	label	definition	parents
```

Comment:
@matentzn Since our recent change, I'm getting outputs now. Some of them have ~13k rows. Is that expected?

I'm still running on some of the ontologies. Will push those files as well soon.

I'm not getting any parents that I'm able to output into the `parents` column that meet the condition (at least that I thought you were asking for). I'll bring this up at the mondo meeting tomorrow.

Reply:
> I'm not getting any parents in any output.

Investigating this is the priority. The code looks ok - unfortunately you will have to debug :/ Sorry for not being helpful on this one!

Reply:
There _may_ be a bug in our logic, I'm not sure. 99%+ of NCIT term objects I instantiated were able to locate their parents using OAK, but when running through our slurp algorithm, no were no parents that met the condition `parent_curie in sssom_object_ids`. 

## Attempt 1: `oaklib.ProntoImplementation`
I think there's a bug. I haven't created an issue yet, though. As mentioned above, NCIT terms were able to locate their parents through `oak`. But this was not the case for the other ontologies.

```py
print(f'{os.path.basename(ontology_path)}: % terms that "have parents": ',
    len([x for x in unmapped_terms if x.direct_parent_curies]) / len(unmapped_terms))
```

Results:
```
doid.owl: % terms that "have parents":  0.0
omim.owl: % terms that "have parents":  0.0
ordo.owl: % terms that "have parents":  0.0
icd10who.owl: % terms that "have parents":  0.0
icd10cm.owl: % terms that "have parents":  0.0
ncit.owl: % terms that "have parents":  0.9979720960415315
```

## Attempt 2: `oaklib.SqlImplementation`
### 2.1: `.outgoing_relationship_map()`
Got `RecursionError`. Made issue: https://github.com/INCATools/ontology-access-kit/issues/290

### 2.2. `.outgoing_relationship_map()`
Haven't looked further into this, so I didn't make an issue.

```
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: edge
[SQL: SELECT edge.subject AS edge_subject, edge.predicate AS edge_predicate, edge.object AS edge_object 
FROM edge 
WHERE edge.object = ?]
[parameters: ('Orphanet:101334',)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
```

### 2.3. `.relationships()`
Same error as (2.2): `sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: edge`

## Attempt 3: SPARQL
Trying now.

Reply:
SPARQL success!
```tsv
mondo_id	xref	label	definition	parents
ID	A oboInOwl:hasDbXref	LABEL	A IAO:0000115	SC %
MONDO:850001	DOID:0040003	benzylpenicillin allergy		MONDO:0000794|MONDO:0000775|MONDO:0005271|MONDO:0005046|MONDO:0021199|MONDO:0000001
MONDO:850002	DOID:0040004	amoxicillin allergy		MONDO:0000794|MONDO:0000775|MONDO:0005271|MONDO:0005046|MONDO:0021199|MONDO:0000001
...
```

Reply:
Everything here is fixed. I just need to move to OAK once it's fixed. We can resolve this thread now.

Reply:
Solved! via `rdfs:subClassOf*` --> `rdfs:subClassOf`